skycultures/arabic/description.en.utf8

```diff
@@ -0,0 +1,296 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+<title>Arabic</title>
+</head>
```

Comment:
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.